### PR TITLE
Shellcheck fixes

### PR DESCRIPTION
**Pull request by roxell**

### CODE REVIEW

**Comment by mwasilew** [comment]
LGTM. Many thanks for fixing these. I'm pretty sure all 'linux' side changes work OK. @chase-qi could you take a look at 'android' tests?

File: automated/android/media-codecs-functional-tests/linaro-android-userspace-tests.sh [hunk 1]
```diff
@@ -1,9 +1,10 @@
-#!/system/bin/sh
+#!/system/bin/bash
```
Comment (liuyq) [inline automated/android/media-codecs-functional-tests/linaro-android-userspace-tests.sh:1]: there is no /system/bin/bash in android

**Comment by chase-qi** [comment]
@mwasilew @roxell LGTM. Thanks for the fixes.

File: automated/android/media-codecs-functional-tests/linaro-android-userspace-tests.sh (hunk 1 above)
Comment (roxell) [reply automated/android/media-codecs-functional-tests/linaro-android-userspace-tests.sh]: fixed

**Pull request merged**